ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/recovery/RecoveryManager.java

```diff
@@ -372,6 +374,25 @@ public static Socket getClientSocket () throws IOException
         return socket;
     }
 
+    /**
+     * @return true if the recovery manager should wait for the object store to recover
+     * certain types {@see getTypeNamesToBlockShutdown} before finally
```

Comment:
We should change this javadoc

Reply:
Thank you! I spotted that there was also a related change needed in PeriodicRecovery. Also it wasn't in the selection box here but was in the Javadoc and changed `Note that this may necessitate the disabling of further transaction creation.` to `Note that this necessitates the disabling of further transaction creation.`